tabpy-server/tabpy_server/common/endpoint_file_mgr.py

```diff
@@ -84,7 +84,7 @@ def cleanup_endpoint_files(name, query_path, retain_versions=None):
     else:
         retain_folders = [os.path.join(local_dir, str(version))
                           for version in retain_versions]
-        logger.info("Retain folder: %s" % retain_folders)
+        logger.log(logging.INFO, f'Retain folders: {retain_folders}')
```

Comment:
Just call `logger.info`

Reply:
If `logger` is a call context logger wrapper it won't have `info` method